Services/JenkinsDocker/JenkinsSF/ApplicationManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
We should probably also include a folder which contains the Jenkins.sh and dockerfile as well. 
Also still wondering if Samples is the right place for this JenkinsDocker. This is something we are planning to support and is not really a sample. I believe we should have a separate repo for this. We can discuss with Mani